libs/types/src/global-types-mappings.ts

```diff
@@ -734,3 +735,23 @@ export const ProposalProductTypeShortName: Record<ProposalProductType, string> =
     SpotProduct: 'Spot',
     PerpetualProduct: 'Perp',
   };
+
+export const LiquidityFeeMethodMapping: { [e in LiquidityFeeMethod]: string } =
+  {
+    /** Fee is set by the market to a constant value irrespective of any liquidity provider's nominated fee */
+    METHOD_CONSTANT: 'Constant',
+    /** Fee is smallest value of all bids, such that liquidity providers with nominated fees less than or equal to this value still have sufficient commitment to fulfil the market's target stake. */
+    METHOD_MARGINAL_COST: 'Marginal cost',
+    METHOD_UNSPECIFIED: 'Unspecified',
+    /** Fee is the weighted average of all liquidity providers' nominated fees, weighted by their commitment */
+    METHOD_WEIGHTED_AVERAGE: 'Weighted average',
+  };
+
+export const LiquidityFeeMethodMappingDescription: {
+  [e in LiquidityFeeMethod]: string;
+} = {
```

Comment:
Could you add a link to the docs here https://docs.vega.xyz/testnet/concepts/liquidity/rewards-penalties#determining-the-liquidity-fee-percentage

Something like "For more detail, visit the docs".

Reply:
added